Automattic/wp-calypso
# WooCommerce Installer: remove unused code

#### Changes proposed in this Pull Request

* Removes most of my-sites/woocommerce, except the new landing page.
* Disables the "start a new store" button and adds a warning message, if the user's email is unverified.

#### Testing instructions



- Test that the new flow starting at /woocommerce-installation/< site id > still works and doesn't throw errors.
- With a user with an unverified email, check that you see the disabled button and warning:
<img width="702" alt="Screen Shot 2022-02-04 at 9 48 40 AM" src="https://user-images.githubusercontent.com/1689238/152549733-55aa82f2-cade-4900-a657-4bdb2bb537b8.png">

- Regression: check that the transfer warning still displays when the site can't be transferred, by applying D71706-code



Related to #60349
